### PR TITLE
Update cron expression

### PR DESCRIPTION
**Pull request by jhou1**
Making the cron expression consistent so that we would be able to use tools to manage the cron in a consistent way.

/assign @liangxia @jianlinliu

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -1602,7 +1602,7 @@ tests:
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-private
 - as: azure-ipi-private-workers-marketimage-p2-f28
-  cron: '@yearly'
+  cron: 30 23 1 4 *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1605]: The `yearly` was done in https://github.com/openshift/release/pull/39542 purposely by @jinyunma

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1605]: Yeah, they are on purpose. The owner of the azure account is asking us to pend the testing on the account to catch up the delayed payment.

Comment (jhou1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1605]: My PR does not work differently, still once every year.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1605]: Ah, got it.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1605]: I'm fine with the changes. I'll let @jinyunma to decide

Comment (jinyunma) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1605]: it's also okay for me.

**Pull request merged**